package.json

```diff
@@ -48,13 +48,14 @@
     "jsdom": "^9.12.0",
     "mocha": "^3.2.0",
     "nodemon": "^1.11.0",
-    "react": "^15.4.2",
-    "react-addons-test-utils": "^15.5.1",
-    "react-dom": "^15.4.2",
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "react-test-renderer": "^15.5.4",
     "sinon": "^2.1.0",
     "sinon-chai": "^2.9.0"
   },
   "dependencies": {
-    "lodash": "^4.17.4"
+    "lodash": "^4.17.4",
+    "prop-types": "^15.5.8"
```

Comment:
Adding as a dependency matches the implementation of react-router-dom, react-transition-group, react-redux, and some other highly used libraries.  This will make it a little more explicit especially for projects utilizing typescript that will most likely not already have prop-types.